### ISSUE TITLE
`ansible-galxy import` fails

### ISSUE THREAD

**Issue by chouseknecht** [ansible/galaxy#1370]
Here's the traceback:

```
ansible-galaxy -vvv import --branch v2.7.1 ansible-network cisco_nxos
ansible-galaxy 2.7.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/Users/chouseknecht/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/chouseknecht/.pyenv/versions/2.7.14/envs/venv27/lib/python2.7/site-packages/ansible
  executable location = /Users/chouseknecht/.pyenv/versions/venv27/bin/ansible-galaxy
  python version = 2.7.14 (default, Nov 14 2017, 23:24:24) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.38)]
Using /etc/ansible/ansible.cfg as config file
Opened /Users/chouseknecht/.ansible_galaxy
https://galaxy.ansible.com/api/v1/imports/
Successfully submitted import request 326840
https://galaxy.ansible.com/api/v1/imports/?id=326840
ERROR! Unexpected Exception, this is probably a bug: 'task_messages'
the full traceback was:

Traceback (most recent call last):
  File "/Users/chouseknecht/.pyenv/versions/venv27/bin/ansible-galaxy", line 118, in <module>
    exit_code = cli.run()
  File "/Users/chouseknecht/.pyenv/versions/2.7.14/envs/venv27/lib/python2.7/site-packages/ansible/cli/galaxy.py", line 162, in run
    self.execute()
  File "/Users/chouseknecht/.pyenv/versions/2.7.14/envs/venv27/lib/python2.7/site-packages/ansible/cli/__init__.py", line 155, in execute
    fn()
  File "/Users/chouseknecht/.pyenv/versions/2.7.14/envs/venv27/lib/python2.7/site-packages/ansible/cli/galaxy.py", line 625, in execute_import
    for msg in task[0]['summary_fields']['task_messages']:
KeyError: 'task_messages'
```

Here's the Ansible version:

```
ansible-galaxy 2.7.0
  config file = /etc/ansible/ansible.cfg
  configured module search path = [u'/Users/chouseknecht/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/chouseknecht/.pyenv/versions/2.7.14/envs/venv27/lib/python2.7/site-packages/ansible
  executable location = /Users/chouseknecht/.pyenv/versions/venv27/bin/ansible-galaxy
  python version = 2.7.14 (default, Nov 14 2017, 23:24:24) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.38)]
```

**Comment by chouseknecht** [comment]
The fix is deployed to `galaxy-dev.ansible.com`. 

Here's some example output:

```
ansible-galaxy -vvv -s https://galaxy-dev.ansible.com import --branch v2.7.1 ansible-network cisco_nxos
ansible-galaxy 2.7.0
  config file = /Users/chouseknecht/ansible.cfg
  configured module search path = [u'/Users/chouseknecht/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/chouseknecht/.pyenv/versions/2.7.14/envs/venv27/lib/python2.7/site-packages/ansible
  executable location = /Users/chouseknecht/.pyenv/versions/venv27/bin/ansible-galaxy
  python version = 2.7.14 (default, Nov 14 2017, 23:24:24) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.38)]
Using /Users/chouseknecht/ansible.cfg as config file
Opened /Users/chouseknecht/.ansible_galaxy
https://galaxy-dev.ansible.com/api/v1/imports/
Successfully submitted import request 280748
https://galaxy-dev.ansible.com/api/v1/imports/?id=280748
Starting import: task_id=280748, repository=ansible-network/cisco_nxos

https://galaxy-dev.ansible.com/api/v1/imports/?id=280748
===== LINTING ROLE =====
yamllint Warnings:
./parser_templates/cli/show_user_account.yaml:62:1: [error] too many blank lines (1 > 0) (empty-lines)
ansible-lint OK.

===== IMPORTING ROLE: cisco_nxos =====
Adding role metadata tags
Adding role platforms
Adding role dependencies

Updating repository versions...
Import completed with 1 warnings and 0 errors
```

**Comment by chouseknecht** [comment]
Fixed and released in 3.1.3
